examples/cloud_run_cloudevents/main.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample creates a function using the CloudEvents SDK
+# (https://github.com/cloudevents/sdk-python)
+import sys
+
+
+def hello(cloudevent):
+    print(f"Received event with ID: {cloudevent['id']}")
```

Comment:
Why prefix with cloud_run? We've established cloudevents are an operating mode of this framework, so this can simply be a cloudevents example.

Reply:
Do you want to remove the cloud_run prefix from every folder? In which case we should do this in a new PR to keep this diff count low.